integration/rabbitmq/consumer/consumer-deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rabbitmq-consumer
+  namespace: rabbits
+type: Opaque
+data:
+  RABBIT_USERNAME: Z3Vlc3Q=
+  RABBIT_PASSWORD: Z3Vlc3Q=
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rabbitmq-consumer
+  namespace: rabbits
+  labels:
+    app: rabbitmq-consumer
+spec:
+  selector:
+    matchLabels:
+      app: rabbitmq-consumer
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: rabbitmq-consumer
+    spec:
+      containers:
+      - name: rabbitmq-consumer
+        image: localhost:5000/rabbit-consumer:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 80
+        env:
+        - name: RABBIT_HOST 
+          value: "rabbitmq-0.rabbitmq.rabbits.svc.cluster.local"
+        - name: RABBIT_PORT
+          value: "5672"
+        - name: RABBIT_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: rabbitmq-publisher
+              key: RABBIT_USERNAME
+        - name: RABBIT_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: rabbitmq-publisher
+              key: RABBIT_PASSWORD
+#---
+#apiVersion: v1
+#kind: Service
+#metadata:
+#  name: rabbitmq-publisher
+#  labels:
+#    app: rabbitmq-publisher
+#spec:
+#  type: LoadBalancer
+#  selector:
+#    app: rabbitmq-consumer
+#  ports:
+#    - protocol: TCP
+#      name: http
+#      port: 80
+#      targetPort: 80
```

Comment:
If not required, please remove

Reply:
Next commit has this commented sections removed in every deployment file